src/plugins/visualizations/public/components/visualization_noresults.tsx

```diff
@@ -44,7 +44,7 @@ export class VisualizationNoResults extends React.Component<VisualizationNoResul
       <div className="visError" ref={this.containerDiv}>
         <div className="item top" />
         <div className="item">
-          <EuiText size="xs" color="subdued">
+          <EuiText size="s" color="subdued">
```

Comment:
Does this one show in the visualization?

Reply:
yes, visualization. I have screenshots in the PR description, here's also another one
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/32652829/26be63d9-7fdf-46d2-bb63-557570461319)

Reply:
Can we keep the text in visualizations small? I know this is subjective, but would prefer not to change content on people's dashboards + don't want to mess with visualizations that might be small and now text will in crease inside them.

Reply:
yeah I have reverted this too